src/nebula-hooks/use-context-menu.ts

```diff
@@ -1,18 +1,17 @@
 // @ts-ignore  TODO: remove tag when onContextMenu is declared in nebula/stardust
 import { onContextMenu } from '@nebula.js/stardust';
 import { copyCellValue } from '../table/utils/accessibility-utils';
-import { Menu } from '../types';
 
 export default function useContextMenu(areBasicFeaturesEnabled: boolean) {
-  onContextMenu?.((menu: Menu, event: any) => {
+  onContextMenu((menu: any, event: any) => {
```

Comment:
put the question mark back

Reply:
As a safeguard if someone does not have the latest nebula version. The table crashes and they get a cryptic error. Better to just skip trying to call `onContextMenu` and the table still runs.